backend/svirt.pm

```diff
@@ -21,13 +21,19 @@ use testapi qw(get_var get_required_var check_var);
 
 use IO::Select;
 
+use Class::Accessor "antlers";
+has instance => (is => "rw", isa => "Num");
+has name     => (is => "rw", isa => "Str");
+
```

Comment:
you don't need it in the constructor. you need it when you call the functions. So I would query the console at that time - and you get the console by its name.